source/libs/page-detect.ts

```diff
@@ -97,6 +97,16 @@ export const isTrending = (): boolean => location.pathname === '/trending' || lo
 
 export const isUserProfile = (): boolean => select.exists('.user-profile-nav');
 
+export const isHistoryPage = (): boolean => {
```

Comment:
These detections are ideally URL-based. It’s best you use `isCommitList` and then detect the required elements in the feature itself.

Reply:
Sure. I will. Maybe we can move it back to `page-detect` if we have another feature for the history pages.

Reply:
No? 😅 No worries. We won't :)